graphene_django/rest_framework/tests/test_mutation.py

```diff
@@ -268,6 +273,34 @@ class Meta:
     assert result.days_since_last_edit == 4
 
 
+def test_perform_mutate_success_with_enum_choice_field():
```

Comment:
I ran this test locally without the new changes in `mutation.py`, and it still passes. 🤔  Do you know why that would be? Does it relate to the version of django-rest-framework perhaps? Is the bug in https://github.com/graphql-python/graphene-django/issues/1385 actually still reproducible/present?

I think it would be worth ensuring that this test fails on master (presumably in the way that https://github.com/graphql-python/graphene-django/issues/1385 described) so that we're confident the fix resolves the original problem. Or if the change in `mutation.py` is no longer needed, we should remove that but still merge your test.

Reply:
@sjdemartini It is still present. I was able to repro with your DRF docs call out..

See: https://github.com/graphql-python/graphene-django/pull/1431#discussion_r1272697792

I wasn't instantiating the `choices` kwarg list correctly.